text/0000-kpack-donation-to-cnb.md

```diff
@@ -0,0 +1,374 @@
+
+[CNB](https://buildpacks.io/) already mentioned [kpack](https://github.com/pivotal/kpack/) in their documentation, specifically, in the tools section. The proposal is:
+
+- Create a new folder name **kpack** inside the [tool](https://github.com/buildpacks/docs/tree/main/content/docs/tools) section in the docs repository
+- Copy kpack's [documentation](https://github.com/pivotal/kpack/tree/main/docs) into this new created folder
+- Update the references and all the required elements to format the documentation according to [CNB](https://buildpacks.io/) site
+
+### Governance
+
+#### Team roles
+
+Based on the [CNB governance policy](https://github.com/buildpacks/community/blob/main/GOVERNANCE.md) and the fact that [kpack](https://github.com/pivotal/kpack/) is a [platform](https://buildpacks.io/docs/concepts/components/platform/) implementation of [Cloud Native Buildpacks](https://buildpacks.io/), it will be added under the responsibility of the [CNB Platform Team](https://github.com/buildpacks/community/blob/main/TEAMS.md#Platform-Team).
+
+How do migrate roles and responsibilities into the CNB governance process?
+
+Currently the [CNB Platform Team](https://github.com/buildpacks/community/blob/main/TEAMS.md#Platform-Team) already has a **team lead** assigned and, by definition, each team can have only one **team lead**. In order to provide the current [kpack](https://github.com/pivotal/kpack/) team with the same accountability for the migrated repositories the proposal is to follow the guidelines describe on the [Component Maintainer Role RFC](https://github.com/buildpacks/rfcs/pull/234)
+
+#### RFC process
+
+Once the migration is completed, [kpack](https://github.com/pivotal/kpack/) will follow the [RFC process](https://github.com/buildpacks/rfcs) stablished in CNB project for any new RFC created in the project.
+
+##### Existing RFC
+
+- **Open**: Currently there are less that 10 [open RFCs](https://github.com/pivotal/kpack/pulls?q=is%3Apr+label%3ARFC+is%3Aopen) (some of them opened 2 years ago) in [kpack](https://github.com/pivotal/kpack/) repository.
+  - The proposal is to suggest the [kpack](https://github.com/pivotal/kpack/) maintainers to:
+    - Triage those RFCs an update their status before the donation.
+    - Co-ordinate the announcement of the donation to the RFCs authors and explain them the strategy after the migration (next section)
+    - After the donation, any open RFCs in [kpack](https://github.com/pivotal/kpack/) repository should be closed
+    - The RFC author should create a new RFC in the CNB RFC [repository](https://github.com/buildpacks/rfcs) and follow the CNB [RFC process](https://github.com/buildpacks/rfcs)
+
+- **Closed**: For historical purpose, we will keep those RFC in the repository.
+
+##### Template
```

Comment:
I don't think that the Existing RFC template is particularly important. I would fine with just completely moving to the CNB template!

Reply:
"Complexity" was copied from an internal VMware RFC process. I don't necessarily know if it is needed in this context.

Reply:
Ok, that's a good reason @matthewmcnew , thanks for the feedback